packages/forms/src/UIForm/fieldsets/Array/Array.component.test.js

```diff
@@ -382,4 +382,36 @@ describe('Array component', () => {
 			]);
 		});
 	});
+
+	describe('#isCloseable', () => {
```

Comment:
You should not test explicitly internal functions.
Instead, test the exposed, api. Here you should use a closeable component and check that the props.isCloseable that is passed is correct.